docs/r2/hyperparameter_tuning_with_hparams.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>provide -&gt; "you'll provide the training function"</p><p><br></p><p>Also, the second sentence is really long for technical doc. Consider breaking it up.</p><p><br></p><p>The training code will look familiar, except you won't hardcode the hyperparameters. Instead, you'll provide a hparams dictionary to the training function ... for a particular training run. You'll then return the model's accuracy.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Addressed.